cuda_bindings/docs/source/install.md

```diff
@@ -27,12 +27,12 @@ Install all optional dependencies with:
 pip install -U cuda-python[all]
 ```
 
-Where the optional dependencies are:
+Where the optional dependencies are now managed through the ``cuda-toolkit`` metapackage for improved version constraints and dependency resolution:
 
-* nvidia-cuda-nvrtc (Provides NVRTC shared library)
-* nvidia-nvjitlink (Provides nvJitLink shared library)
-* nvidia-cuda-nvcc (Provides NVVM shared library)
-* nvidia-cufile (Provides cuFile shared library)
+* ``cuda-toolkit[nvcc,nvrtc,nvjitlink,nvvm]==13.*`` (Provides NVRTC, nvJitLink, and NVVM shared libraries)
+* ``cuda-toolkit[cufile]==13.*; sys_platform == 'linux'`` (Provides cuFile shared library on Linux)
```

Comment:
I don't want to repeat what we do in pyproject.toml. You might be relentless enough to maintain it, but it's a burden for us humans 🥲